src/reports/components/report-sections/full-rule-header.tsx

```diff
@@ -44,21 +43,29 @@ export const FullRuleHeader = NamedFC<FullRuleHeaderProps>('FullRuleHeader', pro
     const renderRuleLink = () => {
         const ruleId = cardResult.id;
         const ruleUrl = cardResult.url;
-        return (
-            <span className="rule-details-id">
-                <NewTabLink
-                    href={ruleUrl}
-                    aria-label={`rule ${ruleId}`}
-                    aria-describedby={ariaDescribedBy}
-                >
-                    {ruleId}
-                </NewTabLink>
-            </span>
+        const displayedRule = ruleUrl ? (
+            <NewTabLink
+                href={ruleUrl}
+                aria-label={`rule ${ruleId}`}
```

Comment:
@karanbirsingh: Regarding your comment on the other PR as to why we have aria-label here and not other cases:

I believe it's due to different designs in the UI. The other case I'd imagine provides context already with the text but this doesn't. That's my guess at least.

Reply:
Got it. At first I was hoping we could share the 'show-text-if-no-link' behavior across both components but in rule-resources we don't show text at all so it seems less likely.